Supported-Fixes.md

```diff
@@ -0,0 +1,812 @@
+# Supported CodeFixes
```

Comment:
👏 👏 👏 👏 👏 👏

Reply:
This is fantastic and we can leverage this within PR’s for upgrade